arkdwallet.Dockerfile

```
@@ -0,0 +1,33 @@
+name: Playwright Tests
```

Hadolint (2.14.0)

[error] unexpected 'n'
expecting '#', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, end of input, or whitespaces

(DL1000)